orchagent/portsorch.cpp

```diff
@@ -128,6 +130,8 @@ PortsOrch::PortsOrch(DBConnector *db, string tableName) :
 
 bool PortsOrch::getPort(string alias, Port &p)
 {
+    SWSS_LOG_ENTER();
+
```

Comment:
no need here as the function returns bool and if it is false it always prints out error messages.